README.md

```diff
@@ -13,6 +13,19 @@ npm install
 npm start
 ```
 
+## Tasks
+
+- `npm start` Run web server.
```

Comment:
We probably need the `npm start` anyway since we have to use this for heroku (as default start task by node instance) We make `npm run dev` for running web server and start watching, but `npm start` only **build** and start **web server**?